src/main/public/stylesheets/components/_report_problem.scss

```diff
@@ -12,9 +13,14 @@ $report-problem-colour: #454a4c;
   color: $report-problem-colour;
   font-size: 16px;
   margin: 90px auto -80px 0;
+  padding: 50px 0 0;
 
   span {
     display: block;
     font-size: 14px;
   }
+
+  a {
```

Comment:
won't this change all links to report-problem-colour? Is that intended?

Reply:
Should be fine as `a` is inside `.report-error` block. It will affect all elements matching `.report-error a`.

Reply:
aah right, didn't see it, had to open above the diff to see it 🙍